soc/arm/atmel_sam0/samd21/soc.h

```diff
@@ -47,14 +47,20 @@
 
 #endif /* _ASMLANGUAGE */
 
+/* Add include for DTS generated information */
+#include <generated_dts_board.h>
```

Comment:
Can you #include <devicetree.h> here? Seems to be the modern file name for DST generated defines

Reply:
I had a look ... I am still running under 2.1.0.  The new API looks cool.  I need to update to 2.2.0 to make the change though.

Reply:
Ok, updated to 2.2.0 and changed this.